### PR TITLE
style: Add code tag and refactor bibtex error messages

### PR DESCRIPTION
**Pull request by DanielPalafox**

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[[QA][my-orcid] No error message when invalid BibTeX file is uploaded](https://trello.com/c/4EYO1gkw/7780-qamy-orcid-no-error-message-when-invalid-bibtex-file-is-uploaded)

File: src/app/record/components/work-stack-group/modals/work-bibtex-modal/work-bibtex-modal.component.ts [hunk 1]
```diff
@@ -48,49 +50,58 @@ export class WorkBibtexModalComponent implements OnInit, OnDestroy {
...
-                  }
-                  that.group[newPutCode] = new FormGroup({
-                    checked: new FormControl(false),
-                  })
-                })
-                that.importForm = new FormGroup(that.group)
+          } else {
+            if (parsed) {
+              const newWorks = []
+              if (parsed?.length === 0) {
```
Comment (leomendoza123) [inline src/app/record/components/work-stack-group/modals/work-bibtex-modal/work-bibtex-modal.component.ts:66]:
```suggestion
              if (parsed.length === 0) {
```

The parse is truthy based on the wrapping `if (parsed)`

File: src/app/record/components/work-stack-group/modals/work-bibtex-modal/work-bibtex-modal.component.ts [hunk 2]
```diff
@@ -48,49 +50,58 @@ export class WorkBibtexModalComponent implements OnInit, OnDestroy {
...
+                that.bibtexErrorNoEntries = true
                 that.loadingWorks = false
-              })
+              }
+              while (parsed?.length > 0) {
+                const cur = parsed.shift()
+                const bibtexEntry = cur.entryType.toLowerCase()
+                if (bibtexEntry !== 'preamble' && bibtexEntry !== 'comment') {
+                  // Filtering @PREAMBLE and @COMMENT
```
Comment (leomendoza123) [inline src/app/record/components/work-stack-group/modals/work-bibtex-modal/work-bibtex-modal.component.ts:74]: a weird comment, probably coming from the old app? 
```suggestion
```

File: src/app/record/components/work-stack-group/modals/work-bibtex-modal/work-bibtex-modal.component.ts [hunk 3]
```diff
@@ -48,49 +50,58 @@ export class WorkBibtexModalComponent implements OnInit, OnDestroy {
...
-                that.importForm = new FormGroup(that.group)
+          } else {
+            if (parsed) {
+              const newWorks = []
+              if (parsed?.length === 0) {
+                that.bibtexErrorNoEntries = true
                 that.loadingWorks = false
-              })
+              }
+              while (parsed?.length > 0) {
```
Comment (leomendoza123) [inline src/app/record/components/work-stack-group/modals/work-bibtex-modal/work-bibtex-modal.component.ts:70]:
```suggestion
              while (parsed.length > 0) {
```

**Review by leomendoza123 — CHANGES_REQUESTED** [review]

**Review by leomendoza123 — APPROVED** [review]

**Pull request merged**